teradatalabs/cdh5-hive-kerberized/files/supervisord.d/hive-server2.conf

```diff
@@ -0,0 +1,8 @@
+[program:hive-server2]
```

Comment:
Regarding your first comment, I guess I can add a new config file for Hive server -- hiveserver2-site.xml file. Not sure if doing `ADD files/conf/hiveserver2-site.xml /etc/hive/conf/hiveserver2-site.xml` would be sufficient to have those changes effective. Will try that out.